### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by ellaboina-vijay**
we added karam and oil and compleate the code please once  review the code

### CODE REVIEW

**Comment by ellaboina-vijay** [comment]
why you added karam

File: readme.MD [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # dosa
 dosa flour added ,
-light oil added ,
```
Comment (ellaboina-vijay) [inline readme.MD:3]: why remove light oil , please conform

Comment (ellaboina-vijay) [reply readme.MD:3]: i added for taste

Comment (ellaboina-vijay) [reply readme.MD:3]: ok

**Review by ellaboina-vijay — COMMENTED** [review]
ok its seem good done

**Review by vijay-ellaboina — APPROVED** [review]

**Pull request merged**